plugin/http/httpstats/definitions.go

```diff
@@ -0,0 +1,126 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package httpstats provides OpenCensus stats instrumentation for the standard library http package.
+package httpstats
+
+import (
+	"fmt"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+)
+
+var (
+	// See: http://unitsofmeasure.org/ucum.html
+	unitByte          = "By"
+	unitDimensionless = "1"
+	unitMillisecond   = "ms"
+
+	rpcBytesBucketBoundaries  = []float64{0, 1024, 2048, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864, 268435456, 1073741824, 4294967296}
+	rpcMillisBucketBoundaries = []float64{0, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000}
+
+	aggCount      = stats.CountAggregation{}
+	aggDistBytes  = stats.DistributionAggregation(rpcBytesBucketBoundaries)
+	aggDistMillis = stats.DistributionAggregation(rpcMillisBucketBoundaries)
+)
+
+var (
+	// ClientRequest is the number of client requests started.
+	ClientRequest = int64Measure("requests", "Number of HTTP requests started", unitDimensionless)
+	// ClientRequestBodySize is the size of request body if set as ContentLength (uncompressed bytes).
+	ClientRequestBodySize = int64Measure("request_size", "HTTP request body size (uncompressed)", unitByte)
+	// ClientResponseBodySize is the size of response body (uncompressed bytes).
+	ClientResponseBodySize = int64Measure("response_size", "HTTP response body size (uncompressed)", unitByte)
+	// ClientLatency is the end-to-end client latency (in milliseconds).
+	ClientLatency = floatMeasure("latency", "End-to-end latency", unitMillisecond)
+
+	// ClientRequestCount is a count of all instrumented HTTP requests.
+	ClientRequestCount = defaultView(ClientRequest, aggCount)
+	// ClientRequestBodySizeDistribution is a view of the size distribution of all instrumented request bodies.
+	ClientRequestBodySizeDistribution = defaultView(ClientRequestBodySize, aggDistBytes)
+	// ClientResponseBodySizeDistribution is a view of the size distribution of instrumented response bodies.
+	ClientResponseBodySizeDistribution = defaultView(ClientResponseBodySize, aggDistBytes)
+	// ClientLatencyDistribution is a view of the latency distribution of all instrumented requests.
+	ClientLatencyDistribution = defaultView(ClientLatency, aggDistMillis)
+	// ClientResponseCountByStatusCode is a view of response counts by StatusCode.
+	ClientResponseCountByStatusCode = mustView(stats.NewView(
+		qualify("request_count_by_method"),
+		"Client request count by HTTP method",
+		[]tag.Key{Method},
+		ClientRequest,
+		aggCount,
+		&stats.Cumulative{}))
+	// ClientRequestCountByMethod is a count of all instrumented HTTP requests by Method.
+	ClientRequestCountByMethod = mustView(stats.NewView(
+		qualify("response_count_by_status_code"),
```

Comment:
Same as above.

Reply:
I am updating the specs. I created those before having any direct experience with instrumenting HTTP so some of the values measures specified are not very easy to get and probably not that useful. Also I changed the naming a bit to bring it closer to what we have for gRPC in Go.

Reply:
https://github.com/census-instrumentation/opencensus-specs/pull/48